app/src/androidTest/java/org/glucosio/android/HelloActivityTest.java

```diff
@@ -89,7 +87,7 @@ public static void clearAppData() {
     }
 
     @Test
-    public void check_001_IfHelloActivityIsCompletelyDisplayed() throws InterruptedException {
+    public void check_001_IfHelloActivityIsCompletelyDisplayed() {
```

Comment:
I would recommend writing in a BDD style to make more sense, ie:
`givenHelloActivityViews_whenCheckShare_thenVerifyPerfomScrollTo`

Reply:
Sorry, I'm not going to do it now. Added ticket #387